pom.xml

```diff
@@ -540,7 +540,7 @@
         <carbon.kernel.registry.imp.pkg.version.range>[1.0.1, 2.0.0)</carbon.kernel.registry.imp.pkg.version.range>
 
         <!-- Carbon Identity Framework version -->
-        <carbon.identity.framework.version>5.7.0</carbon.identity.framework.version>
+        <carbon.identity.framework.version>5.8.0-SNAPSHOT</carbon.identity.framework.version>
```

Comment:
This should be a released version. Snapshot versions are removed when the next version is released.